.github/workflows/CONTRIBUTING.md

```diff
@@ -1,10 +1,73 @@
-These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to
-this document in a pull request.
+## Step 2 : Fork the Project
+- Fork this Repository. This will create a Local Copy of this Repository on your Github Profile. Keep a reference to the original project in `upstream` remote.
+```
+$ git clone https://github.com/<your-username>/cbj_app.git
+$ cd cbj_app
+$ git remote add upstream https://github.com/CyBear-Jinni/cbj_app.git
+```
+- If you have already forked the project, update your copy before working.
+```
+$ git remote update
+$ git checkout <branch-name>
+$ git rebase upstream/<branch-name>
+```
+## Step 3 : Branch
+Create a new branch. Use its name to identify the issue your addressing.
+```
+# It will create a new branch with name Branch_Name and switch to that branch 
+$ git checkout -b branch_name
+```
+## Step 4 : Work on the issue assigned
+- Work on the issue(s) assigned to you. 
+- Add all the files/folders needed.
+- After you've made changes or made your contribution to the project add changes to the branch you've just created by:
+```
+# To add all new files to branch Branch_Name
+$ git add .
+```
 
-## First rule, please open pull request to dev branch and not to master
+## Step 5 : Commit
+- To commit give a descriptive message for the convenience of reveiwer by:
+```
+# This message get associated with all files you have changed
+$ git commit -m 'message
+```
+- **NOTE**: A PR should have only one commit. Multiple commits should be squashed.
+## Step 6 : Work Remotely
+- Now you are ready to your work to the remote repository.
+- Push your work to your remote repository.
+
+- When your work is ready and complies with the project conventions, upload your changes to your fork:
+
+```
+# To push your work to your remote repository
+$ git push -u origin Branch_Name
+```
+
+## Step 7 : Pull Request
+- Go to your repository in browser and click on compare and pull requests. 
+- Add a title and description to your pull request that explains your contribution.
+
+- 🥳:tada: Your Pull Request has been submitted and will be reviewed by the moderators and merged.
+
+## Need more help?🤔
+You can refer to the following articles on basics of Git and Github and also contact the Project Mentors, in case you are stuck:
+- [Forking a Repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+- [Cloning a Repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+- [How to create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github)
+- [Getting started with Git and GitHub](https://towardsdatascience.com/getting-started-with-git-and-github-6fcd0f2d4ac6)
+- [Learn GitHub from Scratch](https://lab.github.com/githubtraining/introduction-to-github)
+
+## Tip from us😇
+Learning and understanding takes time, so don't worry if things seem challenging at first. So, do not worry at all. We know **you have got this**!💪
```

Comment:
Not needed.
You can refer to the wiki for more info 
https://github.com/CyBear-Jinni/cbj_app/wiki
Take a look and decide if it is the right place to add it

Reply:
@guyluz11  As you mentioned, I have made the required changes, and you can check them
Thank you